stats.py

```diff
@@ -47,6 +50,8 @@ def _main():
                 break
 
     for name, _ in RE_DICT:
+        if not name in times.keys():
+            continue
```

Comment:
I think this is strange.

If the `name` is not contained in `times`, that means the script has a bug.

Reply:
The absence of this IF condition forces the results to be parsed to include all libraries defined in `run_all.sh`.
This was inconvenient when testing a part of libraries, so I modified it. But, on second thought, a manual comment out would be sufficient. I reverted it.

Reply:
https://github.com/legalforce-research/tokenizer-speed-bench/pull/2/commits/67131af40a501cb8b5fbfb06ad79ef77de2d197f